inference-engine/src/transformations/src/transformations/serialize.cpp

```diff
@@ -69,16 +75,106 @@ class XmlSerializer : public ngraph::AttributeVisitor {
 public:
     XmlSerializer(pugi::xml_node& data,
                   std::ostream& bin_data,
-                  std::string& node_type_name)
+                  std::string& node_type_name,
+                  const std::map<std::string, ngraph::OpSet>& custom_opsets)
         : m_xml_node(data)
         , m_bin_data(bin_data)
-        , m_node_type_name(node_type_name) {
+        , m_node_type_name(node_type_name)
+        , m_custom_opsets(custom_opsets) {
+    }
+
+    void map_type_from_body(const pugi::xml_node& xml_node,
+        const std::string map_type, std::vector<std::string>& output) {
+        if (!map_type.compare("Result") || !map_type.compare("Parameter")) {
```

Comment:
IMHO, `compare` looks odd. 
-> https://en.cppreference.com/w/cpp/string/basic_string/operator_cmp -> `9` version takes `const char*`

Reply:
right, those comparisons do not look like naturally. Switched to ==.